app/Http/Controllers/TaskController.php

```diff
@@ -177,5 +177,41 @@ public function archived(Request $request)
    
     }
 
-   
+   public function getRecurringTask()
+    {           
+        $tasks = $this->taskService->all();
+
+        $getArchive = [];
+        
+        foreach($tasks as $task)
+        {
+            if(isset($task['recurring'])){
+                 if($taskRecurring["recurring"][0] == "True"){
+                    array_push($getArchive, $task);
+                 }
+            }
+        }
+
+        return response()->json([
+            'message' => "Request Success",
+            'data' => $getArchive
+        ],200);
+    }
+    
+    public function createRecurringTask(Request $request, $id)
```

Comment:
The recurring task field is supposed to be a json which hold some data like when you want the task should keep recurring check the data structure [here](https://docs.google.com/document/d/1ncSNKNsVrd5fLekRpoy2TQYeB-J3g81bFw9poOtO6fU/edit)

Reply:
But I don't see the data there.